packages/cloud2edge/requirements.yaml

```diff
@@ -12,8 +12,8 @@
 #
 dependencies:
   - name: hono
-    version: ^1.10.18
+    version: ~2.1.0
```

Comment:
Can we change that to `^2.1.0` to also allow future minor versions?

Reply:
done